NohanaImagePicker.xcodeproj/project.pbxproj

```diff
@@ -170,6 +171,14 @@
 /* End PBXFrameworksBuildPhase section */
 
 /* Begin PBXGroup section */
+		0318909429529D770094438E /* NohanaImagePickerTests */ = {
```

Comment:
Please Remove unused group.

Reply:
@tyokujin 
I've done removing the unused group.